apps/whale-api/src/module.api/stats.controller.ts

```diff
@@ -96,6 +104,20 @@ export class StatsController {
     }
   }
 
+  @Get('/rewards/distribution')
+  async getRewardDistribution (
+    @Query('height', ParseIntPipe) reductionHeight = 0
+  ): Promise<RewardDistributionData> {
+    const options: CoinbaseSubsidyOptions = this.network === 'mainnet'
+      ? MainNetCoinbaseSubsidyOptions
+      : TestNetCoinbaseSubsidyOptions
```

Comment:
I think we can set `options` once in the constructor so that we don't have to do repeat it for every request